saskatoon/harvest/admin.py

```diff
@@ -69,25 +71,23 @@ class PropertyAdmin(LeafletGeoAdmin):
     model = Property
     inlines = [PropertyImageInline]
     list_display = (
-        '__str__',
-        'authorized',
+        'short_address',
+        'owner_edit',
+        'owner_type',
+        'owner_phone',
+        'owner_email',
         'pending',
-        'harvest_every_year', 
+        'authorized',
         'approximative_maturity_date',
-        'avg_nb_required_pickers',
         'neighborhood',
         'city',
         'postal_code',
+        'pk'
```

Comment:
What is the data that this field represent? 'pk' is not very verbose...

Reply:
pk is the Primary Key